lib/cam_code_scanner.dart

```diff
@@ -142,43 +142,43 @@ class _CamCodeScannerState extends State<CamCodeScanner> {
-                      Positioned(
-                        top: (widget.height / 2) - (widget.height * .1),
-                        left: (widget.width * .1),
-                        child: Container(
-                          decoration: BoxDecoration(
-                            border: Border.all(
-                              color: Colors.green,
-                              width: 1,
-                            ),
-                          ),
-                          child: SizedBox(
-                            width: widget.width * .8,
-                            height: widget.height * .2,
-                            child: widget.showDebugFrames
-                                ? _imageWidget
-                                : Container(),
-                          ),
-                        ),
-                      ),
-                      Center(
-                        child: CustomPaint(
-                          size: Size(
-                            widget.width * .5,
-                            widget.height * .2,
-                          ),
-                          painter: _ScannerLine(
-                            color: Colors.red,
-                          ),
-                        ),
-                      ),
+                      // if (widget.showDebugFrames)
+                      //   Container(
+                      //     width: widget.width,
+                      //     height: widget.height,
+                      //     color: Colors.black.withOpacity(0.8),
+                      //     child: Text(''),
+                      //   ),
+                      // Positioned(
+                      //   top: (widget.height / 2) - (widget.height * .1),
+                      //   left: (widget.width * .1),
+                      //   child: Container(
+                      //     decoration: BoxDecoration(
+                      //       border: Border.all(
+                      //         color: Colors.green,
+                      //         width: 1,
+                      //       ),
+                      //     ),
+                      //     child: SizedBox(
+                      //       width: widget.width * .8,
+                      //       height: widget.height * .2,
+                      //       child: widget.showDebugFrames
+                      //           ? _imageWidget
+                      //           : Container(),
+                      //     ),
+                      //   ),
+                      // ),
+                      //Center(
+                      //  child: CustomPaint(
+                      //    size: Size(
+                      //      widget.width * .5,
+                      //      widget.height * .2,
+                      //    ),
+                      //    painter: _ScannerLine(
+                      //      color: Colors.red,
+                      //    ),
+                      //  ),
+                      //),
```

Comment:
we should extract all this code to a separated widget in order to improve code readability and also to toggle the overlay visibility much easier too